frontend/src/Components/HomePage.js

```diff
@@ -32,9 +32,25 @@ class HomePage extends React.Component {
             return this.setState({ error: 'Password is required' });
         }
 
-        return this.setState({ error: '' });
+        /* Send username and password to backend */
```

Comment:
There's a file in the frontend that has the basic call to the backend (serviceCalls.js). I can change the call over there and add the method/headers/body changes in my PR so there's no merge conflict or anything.